### PR TITLE
add maven & fix broken maven in requiredSdk guide

### PR DESCRIPTION
**Pull request by EbonyLouis**
Reformatting the code snippets, adding new snippets, and removing unnecessary imports.

### CODE REVIEW

File: site/docs/tbdex/issuer/vc-issuance.mdx [hunk 1]
```diff
@@ -23,14 +23,19 @@ If you need to create a Credential Issuance app from scratch, continue on with t
 
 <LanguageSwitchBlock>
   <div language="Kotlin">
-    * Create a web server using <a href="https://www.jetbrains.com/help/idea/ktor.html#run_ktor_app">Ktor</a> that will serve as an API endpoint.
+    * Create a web server using{' '}
```
Comment (angiejones) [inline site/docs/tbdex/issuer/vc-issuance.mdx:26]: how come you changed this? it now looks like this

image

File: site/docs/tbdex/issuer/vc-issuance.mdx [hunk 2]
```diff
@@ -23,14 +23,19 @@ If you need to create a Credential Issuance app from scratch, continue on with t
...
   </div>
   <div language="JavaScript">
-    * Create a web server using <a href="https://expressjs.com/">Express</a> that will serve as an API endpoint.
+    * Create a web server using <a href="https://expressjs.com/">Express</a>{' '}
+    that will serve as an API endpoint.
   </div>
 </LanguageSwitchBlock>
-* Design the credential you'd like to offer.
-* Implement an issuance path where you'll accept a request for issuance, perform any necessary checks, and issue the credential.
+* Design the credential you'd like to offer. * Implement an issuance path where you'll
```
Comment (angiejones) [inline site/docs/tbdex/issuer/vc-issuance.mdx:37]: image

File: site/docs/tbdex/pfi/required-sdks.mdx [hunk 3]
```diff
@@ -4,46 +4,51 @@ title: Required SDKs
...
 
 ## Import Classes
 
 To send and receive tbDEX messages, import the following classes:
 
-<Shnip snippets={[
-    { snippetContent: importJs, language: 'JavaScript'},
-  ]}
+<Shnip
+  snippets={[{ snippetContent: importJs, language: 'JavaScript' }]}
```
Comment (angiejones) [inline site/docs/tbdex/pfi/required-sdks.mdx:51]: can this be changed to `snippetName` and remove the import?

**Review by angiejones — CHANGES_REQUESTED** [review]

File: site/docs/tbdex/issuer/vc-issuance.mdx (hunk 1 above)
Comment (EbonyLouis) [reply site/docs/tbdex/issuer/vc-issuance.mdx:26]: hmm I didn't make changes there 🤔

File: site/docs/tbdex/issuer/vc-issuance.mdx (hunk 2 above)
Comment (EbonyLouis) [reply site/docs/tbdex/issuer/vc-issuance.mdx:37]: I wonder when I saved the file if my IDE did formatting or something, I'll fix this now

File: site/docs/tbdex/issuer/vc-issuance.mdx [hunk 4]
```diff
@@ -23,13 +23,13 @@ If you need to create a Credential Issuance app from scratch, continue on with t
 
 <LanguageSwitchBlock>
   <div language="Kotlin">
-    * Create a web server using <a href="https://www.jetbrains.com/help/idea/ktor.html#run_ktor_app">Ktor</a> that will serve as an API endpoint.
+    * Create a web server using <a href="https://www.jetbrains.com/help/idea/ktor.html#run_ktor_app">Ktor </a> that will serve as an API endpoint.
```
Comment (angiejones) [inline site/docs/tbdex/issuer/vc-issuance.mdx:26]:
```suggestion
    * Create a web server using <a href="https://www.jetbrains.com/help/idea/ktor.html#run_ktor_app">Ktor</a> that will serve as an API endpoint.
```

**Review by angiejones — APPROVED** [review]

**Pull request merged**